svc-managers/managers/common_test.go

```diff
@@ -132,7 +132,7 @@ func mockGetPluginData(pluginID string) (mgrmodel.Plugin, *errors.Error) {
 	}, nil
 }
 
-func mockUpdateManagersData(key string, managerData map[string]interface{}) error {
+func mockUpdateData(key string, managerData map[string]interface{}, table string) error {
```

Comment:
How do we verify this action later ?

Reply:
There is a separate test for UpdateData however in this code it is only mocking the response